olwebsite-app/src/components/SearchBox/SearchBox.jsx

```diff
@@ -0,0 +1,49 @@
+import React, {useState} from "react";
+import PropTypes from "prop-types";
+import { FaSearch } from "react-icons/fa";
+
+const SearchBox = ({ onSearch, placeholder }) => {
+
+  // If we decide to search on change, raise the state
+  const [searchQuery, setSearchQuery] = useState("");
+
+  const handleKeyPress = key => {
+      if(key === "Enter") {
+          onSearch(searchQuery);
+      }
+  }
+
+  /************************************
+   * Render
+   ************************************/
+
+  return (
+    <div className="search-box">
+      <div className="search-box-input-wrapper">
+        <FaSearch 
```

Comment:
can you put a tabIndex on this Icon so we can tab to it (508 compliance)

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
Good catch.